operator/roles/ansible-service-broker/tasks/main.yml

```diff
@@ -131,6 +133,15 @@
     - reconciled_generation != ''
     - pod
 
+- name: Delete the old trusted-ca-bundle if it exists
+  k8s:
+    state: absent
```

Comment:
I assume that the `state: absent` tells the module to ensure that the ConfigMap should be "absent". Is that how I should read that?

Reply:
That is correct